tracee-ebpf/tracee/tracee.go

```diff
@@ -1821,6 +1830,19 @@ func (t *Tracee) processNetEvents() {
 						pkt.TcpOldState,
 						pkt.TcpNewState,
 						pkt.SockPtr)
+				case DebugNetTcpConnect:
+					fmt.Printf("%v  %-16s  %-7d  debug_net/tcp_connect  LocalIP: %v, LocalPort: %d, RemoteIP: %v, RemotePort: %d, Protocol: %d, OldState: %d, NewState: %d, SockPtr: 0x%x\n",
+						timeStampObj,
+						comm,
+						hostTid,
+						netaddr.IPFrom16(pkt.LocalIP),
+						pkt.LocalPort,
+						netaddr.IPFrom16(pkt.RemoteIP),
+						pkt.RemotePort,
+						pkt.Protocol,
+						pkt.TcpOldState,
+						pkt.TcpNewState,
+						pkt.SockPtr)
```

Comment:
These lines are not relevant for this debug message.
You can use the same arguemtns as in DebugNetUdpDestroySock instead

Reply:
you're right, fixed it.